satpy/aux_download.py

```diff
@@ -198,6 +198,17 @@ def _find_registerable_files_compositors(sensors=None):
         sensors = composite_loader.all_composite_sensors()
     if sensors:
         composite_loader.load_compositors(sensors)
+    _register_modifier_files(composite_loader)
+
+
+def _register_modifier_files(composite_loader):
+    for mod_sensor_dict in composite_loader.modifiers.values():
+        for mod_name, (mod_cls, mod_props) in mod_sensor_dict.items():
+            try:
+                mod_cls(**mod_props)
```

Comment:
do we have to instantiate the modifier to register it? It looks like the instance is then thrown away. Could we have rather eg a classmethod to call for registering the files?

Reply:
Yes and no. I could require that modifiers who use this make everything available from a class method, but then that changes how and what they do in the `__init__` of the modifier. Or it might result in duplicated code. This seemed to be the easiest solution at the time.